### ISSUE TITLE
The game crashes

### ISSUE THREAD

**Issue by Felipefpl** [tipun81/tipun_gui_mod#6]
@tipun81 - hi, i downloaded the latest release from here, my game is patched with dlc-merger and no other mods are installed, i got the 3 errors below:
![1](https://github.com/tipun81/tipun_gui_mod/assets/5569804/5798740c-5c40-464d-b27d-c11cf9c3410a)
![2](https://github.com/tipun81/tipun_gui_mod/assets/5569804/1451f142-5168-4497-97cf-dba02be627d7)
![3](https://github.com/tipun81/tipun_gui_mod/assets/5569804/f14b1dbd-9d31-43e8-aebc-a0e7a9bc78be)

**Comment by Lzw104522773** [comment]
Yes, I also encountered these three similar prompts. I installed them on the Chinese versions of BG1EE and BG2EE, and BG2EE crashes when opening the game
